client/state/selectors/get-jetpack-scan-state.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import 'state/data-layer/wpcom/sites/scan';
+
+/**
+ * Returns the current Jetpack Scan state for a given site.
+ * Returns undefined if the site is unknown, or if no information is available.
+ *
+ * @param {object} state	Global state tree
+ * @param {number} siteId	The ID of the site we're querying
+ * @returns {?object}		An object describing the current scan state
+ */
+export default function getJetpackScanState( state, siteId ) {
```

Comment:
I think a better name for this selector would be `getSiteScanState` as something similar was [recommend to me here ](https://github.com/Automattic/wp-calypso/pull/41111#discussion_r408826291).